### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by jdufresne**
Discovered using the command:

    flake8 --select F401 .

### CODE REVIEW

File: examples/statemachine/documentSignoffDemo.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 # Example of a state machine modeling the state of a document in a document
 # control system, using named state transitions
 #
-import statemachine
```
Comment (paulmcg-iio) [inline examples/statemachine/documentSignoffDemo.py]: Even though there are no references to the statemachine module, it must be imported in order for the .pystate modules to be importable.

**Comment by jdufresne** [comment]
@paulmcg-iio did you remove your previous comment? Does it no longer apply or should I go ahead and revert the `statemachine` imports?

**Comment by ptmcg** [comment]
I removed the comment because I accidentally used my work GH account instead of my personal account. The body of the comment is still valid - all the statemachine demos need to `import statemachine`, which will define the import hooks to support importing `.pystate` modules.

**Comment by jdufresne** [comment]
Thanks! I have restored the `statemachine` imports in the latest revision.

**Pull request merged**